Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM frolvlad/alpine-glibc:alpine-3.12
```

Comment:
This variant of alpine is necessary because GraalVM dynamically links against glibc, but alpine doesn't include glibc.